docs/local-setup/running-testnet.md

```diff
@@ -141,18 +134,25 @@ sudo apt install -y git binutils-dev libcurl4-openssl-dev zlib1g-dev libdw-dev l
 
 ## Success Message
 
-Once you have followed either of the above steps, you should see a success message that looks similar to either of the messages below:
+Once you have followed the steps for running a node with Docker or of Compiling without Docker, you should see messages similar to as shown below:
+
 
-Creating genesis
-**Genesis created**
-Starting NEAR client and Watchtower dockers...
+```text
+Using local binary at path/to/nearcore/target/{debug, release}
```

Comment:
here brace expansion seems fine since the user is expected to visually pattern match on the text instead of executing it.  but for consistency it may still be better to choose one specific scenario

Reply:
removed braces from all the commands